src/reducers/requests/roles.js

```diff
@@ -20,39 +20,7 @@ function getRolesByNames(state: RequestStatusType = initialRequestState(), actio
     );
 }
 
-function getRoleByName(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
-    return handleRequest(
-        RoleTypes.ROLE_BY_NAME_REQUEST,
-        RoleTypes.ROLE_BY_NAME_SUCCESS,
-        RoleTypes.ROLE_BY_NAME_FAILURE,
-        state,
-        action
-    );
-}
-
-function getRole(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
-    return handleRequest(
-        RoleTypes.ROLE_BY_ID_REQUEST,
-        RoleTypes.ROLE_BY_ID_SUCCESS,
-        RoleTypes.ROLE_BY_ID_FAILURE,
-        state,
-        action
-    );
-}
-
-function editRole(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
-    return handleRequest(
-        RoleTypes.EDIT_ROLE_REQUEST,
-        RoleTypes.EDIT_ROLE_SUCCESS,
-        RoleTypes.EDIT_ROLE_FAILURE,
-        state,
-        action
-    );
-}
-
 export default (combineReducers({
     getRolesByNames,
```

Comment:
Upon making changes to the webapp this entire file can be removed along with the references to the file